src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -140,6 +141,15 @@ private string GetInvocationId()
             return invocationId.ToString();
         }
 
+        public override Task InvokeAllExceptAsync(string methodName, object[] args, IReadOnlyList<string> excludedIds)
+        {
+            return InvokeAllWhere(methodName, args, connection =>
+            {
+                var excludedIdsSet = new HashSet<string>(excludedIds);
```

Comment:
Yeah, this will create a new HashSet for every check. If we decide we need a HashSet for optimization, we would want to initialize it outside the lambda so there's only one instance.